xbmc/epg/EpgDatabase.cpp

```diff
@@ -99,6 +105,17 @@ void CEpgDatabase::UpdateTables(int iVersion)
 
   if (iVersion < 9)
     m_pDS->exec("ALTER TABLE epgtags ADD sIconPath varchar(255);");
+  
+  if (iVersion < 10)
```

Comment:
WTF grrrr I hate sqlite and its limited SQL and nobody cares about at review. Will push a fix for both asap. Thanks for pointing this out.